### PR TITLE
NIFI-4824 - Allow user to specific host ports on Docker Image startup

### PR DESCRIPTION
**Pull request by Chaffelson**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [X] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [X] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [X] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?
No, but that's because one commit is a bug fix, and the other is an enhancement.

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [X] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-docker/dockerhub/README.md [hunk 1]
```diff
@@ -46,12 +46,22 @@ The configuration scripts are suitable for at least 1.4.0+.
...
+
+You can also pass in environment variables to change the NiFi communication ports and hostname using the Docker '-e' switch as follows:
+
+    docker run --name nifi \
+      -p 9090:9090 \
+      -d \
+      -e NIFI_WEB_HTTP_PORT='9090'
+      apache/nifi:latest
+      
+For a list of the environment variables recognised in this build, look into the .sh/secure.sh and .sh/start.sh scripts
```
Comment (jdye64) [inline nifi-docker/dockerhub/README.md]: Enjoy the English touch with "recognised" instead of "recognized" =)

**Comment by jdye64** [comment]
+1 everything built and ran well for me. Merging now.

**Comment by Chaffelson** [comment]
Thanks @jdye64 I think this is my first official contribution too

**Comment by jdye64** [comment]
Alright!! Congratulations! Thank you for the effort and keep up the good work!

**Pull request closed without merging**